### PR TITLE
Kdesktop 1376 paused syncs start at app s start

### PR DESCRIPTION
**Pull request by ChristopheLarchier**

### CODE REVIEW

**Review by luc-guyot-infomaniak — COMMENTED** [review]
Looks good to me. Just waiting for the Windows build to be green (testworkers.cpp(210) : error : Assertion).

**Review by luc-guyot-infomaniak — DISMISSED** [review]

File: src/server/socketapisocket_mac.mm [hunk 1]
```diff
@@ -174,28 +177,36 @@ - (void)connectToLoginAgent {
...
-        return;
-    }
-
-    /*
-    // To debug with an existing login item agent
-    _loginItemAgentConnection = [[NSXPCConnection alloc] initWithMachServiceName:loginItemAgentMachName options:0];
-    if (_loginItemAgentConnection == nil) {
-        NSLog(@"[KD] Failed to connect to login item agent");
-        return;
+    NSString *loginItemAgentMachName;
```
Comment (ClementKunz) [inline src/server/socketapisocket_mac.mm:180]: Perhaps initialize this value to `nil` and check its validity after the if statement

File: src/server/vfs/mac/litesyncextconnector.mm [hunk 2]
```diff
@@ -217,12 +217,19 @@ - (BOOL)connectToExt {
     // Setup connection with LiteSync extension
     NSLog(@"[KD] Setup connection with LiteSync extension");
 
-    // Read LiteSyncExtMachName from plist
-    NSBundle *appBundle = [NSBundle mainBundle];
-    NSString *liteSyncExtMachName = [appBundle objectForInfoDictionaryKey:@"LiteSyncExtMachName"];
-    if (!liteSyncExtMachName) {
-        NSLog(@"[KD] LiteSyncExtMachName undefined");
-        return FALSE;
+    NSString *liteSyncExtMachName;
```
Comment (ClementKunz) [inline src/server/vfs/mac/litesyncextconnector.mm:220]: same?

File: src/server/socketapisocket_mac.mm (hunk 1 above)
Comment (ChristopheLarchier) [reply src/server/socketapisocket_mac.mm]: Done

File: src/server/vfs/mac/litesyncextconnector.mm (hunk 2 above)
Comment (ChristopheLarchier) [reply src/server/vfs/mac/litesyncextconnector.mm]: Done

**Pull request merged**